apps/aevm/src/aevm_eeevm.erl

```diff
@@ -425,6 +425,14 @@ loop(CP, StateIn) ->
                     <<Val:256/integer-unsigned>> = Hash,
                     State4 = push(Val, State3),
                     next_instruction(CP, State, State4);
+                %% 2f: Non-standard Env information
+                ?CREATOR ->
+                    %% 0x2f Creator δ=0 α=1
+                    %% Get address of contract creator.
+                    %% µ's[0] ≡ Ia
```

Comment:
If so, what is difference with `0x30` ?

Reply:
This is the account that created the contract, not the contract itself.

Reply:
What I mean is that this line does not define the semantics of the instruction, and is rather confusing. I suggest removing it.
```suggestion
```